src/hooks/context/models.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { SpecificationFile } from '../validation';
+import { CONTEXTFILE_PATH, ALLOWED_SPECFILE_NAME } from './constants';
+
+
+export interface Contexts {
+	[name: string]: string
+}
+
+export interface ContextContent {
+	_current: string,
+	_contexts: Contexts
+}
+
+export class ContextFile {
```

Comment:
Maybe I would rename this class as StoragedContexts because reading it right now, for me it's like the file that represents a single context. For example, context X that represents the file path/asyncapi.yml 🤷🏻‍♂️

Reply:
today I changed most of the service files, will keep in mind.